### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by neilzone**
typos :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ npm start
 You can now edit the files in the `docs` folder and the site will reflect the changes immediately thanks to
 live reloading.
 
-## Ccontributing
+## contributing
```
Comment (luixxiul) [inline README.md:20]: With `C` 😉

Comment (neilzone) [reply README.md:20]: Aargh

Comment (saghul) [inline README.md:20]:
```suggestion
## Contributing
```

**Comment by saghul** [comment]
Cheers!

**Pull request merged**